kubernetes-client/src/main/java/io/fabric8/kubernetes/client/utils/HttpClientUtils.java

```diff
@@ -139,6 +139,16 @@ public Response intercept(Chain chain) throws IOException {
                 }
             }
 
+            if (config.getUserAgent() != null && !config.getUserAgent().isEmpty()) {
+                httpClient.networkInterceptors().add(new Interceptor() {
+                    @Override
+                    public Response intercept(Chain chain) throws IOException {
+                        Request agent = chain.request().newBuilder().addHeader("User-Agent", config.getUserAgent()).build();
```

Comment:
This should just use `header` target than `addHeader` to ensure only single value for the header. Sorry didn't notice that before.